web3.json.pas

```diff
@@ -210,9 +210,14 @@ function getPropAsUInt64(const obj: TJsonValue; const name: string; const def: U
   const P = TJsonObject(obj).Get(name);
   if Assigned(P) and Assigned(P.JsonValue) then
     if P.JsonValue is TJsonNumber then
-      Result := TJsonNumber(P.JsonValue).AsUInt64
+{$IF CompilerVersion < 40.0}
+      Result := TJsonNumber(P.JsonValue).AsInt64
+{$IFEND}
+{$IF CompilerVersion >= 40.0}
+        Result := TJsonNumber(P.JsonValue).AsUInt64
+{$IFEND}
     else if P.JsonValue is TJsonString then
-      Result := StrToUInt64Def(TJsonString(P.JsonValue).Value, def);
+      Result := StrToUInt64Def(TJsonString(P.JsonValue).value, def);
```

Comment:
Why did the case of this property got changed?

Reply:
I'm unsure why `Value` (capitalized) got changed into `value` (lowercase).

Reply:
it was me using Ctrl + D.
I have removed that.